ws/ws-tests.ts

```diff
@@ -55,6 +56,14 @@ var WebSocketServer = WebSocket.Server;
 }
 
 {
+    var httpsserver = https.createServer({});
+    var wss = new WebSocket.Server({ server: httpsserver});
+    var httpserver = http.createServer();
+    var wss = new WebSocket.Server({ server: httpserver});
+}
```

Comment:
??? what are the differences to what I commited.

expected of the camel case?

or the non instantiate of the Server type.

... this test is only for the type not for function.

your second would be mime favorite. I will change it soon.

meno

On Wed, Sep 14, 2016, 09:37 TonyYang <notifications@github.com> wrote:

> In ws/ws-tests.ts
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11213#discussion_r78698941>
> :
>
> > @@ -55,6 +56,14 @@ var WebSocketServer = WebSocket.Server;
> >  }
> >
> >  {
> > +    var httpsserver = https.createServer({});
> > +    var wss = new WebSocket.Server({ server: httpsserver});
> > +    var httpserver = http.createServer();
> > +    var wss = new WebSocket.Server({ server: httpserver});
> > +}
>
> Personally, I suggest:
>
> // Test that server implements http.Server or https.Server
> var httpsServer = https.createServer();
> var wss = new WebSocket.Server({ server: httpsServer});
> var httpServer = http.createServer();
> var wss = new WebSocket.Server({ server: httpServer});
>
> or
>
> // Test that server implements http.Server or https.Server
> var httpsServer: https.Server;
> var wss = new WebSocket.Server({ server: httpsServer});
> var httpServer:  http.Server;
> var wss = new WebSocket.Server({ server: httpServer});
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11213/files/8ee8dac28dd747f6b8a0ade9827fa39e69d14e6a#r78698941>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABLJsMvCnCvR4RiCyuKRBbmdzxdHaCzks5qp6QbgaJpZM4J8RfN>
> .
>